endpoint_handlers.go

```diff
@@ -148,7 +141,7 @@ func EndpointCreate(c echo.Context) error {
 
 	err = service.ValidateEndpointCreateRequest(endpointDao, input)
 	if err != nil {
-		return c.JSON(http.StatusBadRequest, util.ErrorDoc(fmt.Sprintf("Validation failed: %s", err.Error()), "400"))
+		return util.NewErrBadRequest(fmt.Sprintf("Validation failed: %s", err.Error()))
```

Comment:
You don't need to call the `.Error()` function when using the `fmt` package's functions: https://go.dev/tour/methods/19

Reply:
good eye 😁  thanks 😁  changed